ro_help/hub/admin.py

```diff
@@ -85,6 +98,90 @@ def get_readonly_fields(self, request, obj=None):
 
         return []
 
+    def save_model(self, request, obj, form, change):
+        super().save_model(request, obj, form, change)
+        if obj.accepts_transfer:
+            if not obj.accounts.all():
+                self.message_user(request, 'Trebuie sa adaugati cel putin un cont pentru a accepta transfer bancar', level=messages.ERROR)
```

Comment:
Let's add a `transaction.atomic()` on this function.

Reply:
The transaction should be on the NGO model, right?